mvc_flask/middleware/spoofing.py

```diff
@@ -0,0 +1,46 @@
+from flask import Request
+from werkzeug.formparser import parse_form_data
+
+
+class InputProcessed:
+    def read(self, *args):
+        raise EOFError(
```

Comment:
The `request.json`, `request.get_json()`, `request.data`, `request.get_data()` methods and attributes are raising this exception.

#### Reproduction:
Change the create method of MessagesController

```py
# tests/app/controllers/messages_controller.py

    def create(self):
        db.session.add(Message(**request.json))
        db.session.commit()
        return {}, 201
```

Curl: 
```bash
curl -L http://localhost:5000/messages --data-raw '{"title": "fuba"}'
```

Reply:
Thanks @taconi , I go analyze

Reply:
@taconi sorry, I took longer than expected. Can you test again? Thanks

Reply:
I tested create and it's working, I saw the tests you added too, great